src/main/webapp/js/enroll.js

```diff
@@ -0,0 +1,89 @@
+
+$(document).ready(function () {
```

Comment:
Detailed comments in this file would be helpful.

Reply:
Added